src/containers/Navbar.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const Navbar = () => {
+  return (
+    <>
+      <i className="fa fa-2x fa-refresh" aria-hidden="true"></i>
```

Comment:
I think you can replace fa with component (use a lib or svg icon...)

Reply:
ok done